nav2_map_server/launch/map_saver_server.launch.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2020 Samsung R&D Institute Russia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from ament_index_python.packages import get_package_share_directory
+
+from launch import LaunchDescription
+import launch_ros.actions
+
+def generate_launch_description():
+    # Parameters
+    lifecycle_nodes = ['map_saver_server']
+    use_sim_time = True
+    autostart = True
+    save_map_timeout = 5000
```

Comment:
Lets make this timeout more realistic, `2000`?

Reply:
Done: https://github.com/ros-planning/navigation2/commit/84cff7d8cf43443e23bc4852129791d665f6e5c7#diff-3fc153ccd4ba9fe4cdeaf6c7930d9b4fR29